### DISCUSSION TITLE
Is it possible for the focus to follow only when the cursor is moved from one display to another

### DISCUSSION THREAD

**Discussion by 0xSmit** [asmvik/yabai#698] [Ideas]
I've been using yabai to follow focus on mouse movements, however my main aim is to transfer focus to the screen the mouse moves and not transfer focus to every app the cursor moves

**Comment by asmvik** [comment]
You can do this by editing the source, but I don't feel like including this as a separate config option.
What you want to do is basically get rid of this `if` statement, and keep *only* the `else` part: https://github.com/koekeishiya/yabai/blob/master/src/event.c#L798

**Comment by carlosflorencio** [comment]
Would also benefit from this if it was a dedicated config :).
